routing-daemon/lib/openshift/routing/models/f5-icontrol-rest.rb

```diff
@@ -249,26 +274,40 @@ def initialize logger, cfgfile
 
       read_config cfgfile
 
-      # Create the openshift_application_routes an openshift_application_aliases
-      # policies if they do not already exist.  The create_route and
-      # delete_route methods add and delete the application-specific rules to
-      # and from the former, and the add_pool_alias and delete_pool_alias
-      # methods add and delete the application-specific rules to and from the
-      # latter.
-      ["openshift_application_routes","openshift_application_aliases"].each do |policy|
-        begin
-          get(url: "https://#{@host}/mgmt/tm/ltm/policy/#{policy}")
-        rescue RestClient::ResourceNotFound
-          @logger.info "No #{policy} policy exists.  Creating..."
-          post(url: "https://#{@host}/mgmt/tm/ltm/policy",
-               payload: {
-                 "kind" => "tm:ltm:policy:policystate",
-                 "name" => policy,
-                 "controls" => ["forwarding"],
-                 "requires" => ["http"],
-                 "strategy" => "first-match",
-               }.to_json)
-        end
+      # Create the openshift_application_aliases policy if it does not already
+      # exist. The add_pool_alias and delete_pool_alias methods add and delete
+      # the application-specific rules to and from the latter.
+      policy = 'openshift_application_aliases'
```

Comment:
We should either use this variable below where we now hardcode `openshift_application_aliases` or eliminate this variable and hardcode `openshift_application_aliases` everywhere.